package.json

```diff
@@ -23,7 +23,7 @@
     "version": "grunt build && grunt copyForPublish && git add package.json release/* ",
     "postversion": "export GITTAG=\"echo $(git describe --abbrev=0 --tags | sed 's/^v//')\" && git push --force --set-upstream origin bumpVersion --follow-tags && git checkout gh-pages && git pull && cp out/connect-rtc-debug.js ./connect-rtc-debug-`$GITTAG`.js && cp out/connect-rtc.min.js ./connect-rtc-`$GITTAG`.min.js && ln -fs connect-rtc-debug-`$GITTAG`.js connect-rtc-debug-latest.js && ln -fs connect-rtc-`$GITTAG`.min.js connect-rtc-latest.min.js && git add connect-rtc*.js && git commit -m `$GITTAG` && git push --set-upstream origin gh-pages && git checkout master",
     "prepublish": "grunt build",
-    "test": "grunt && mocha --compilers js:babel-core/register test/unit"
+    "test": "grunt && nyc --reporter=html mocha --compilers js:babel-core/register test/unit"
```

Comment:
Do we need to add this to any of the above scripts? Right now I see it added only to preversion script as dependent build script that needs to be run.

Reply:
Discussed offline.

Reply:
>If preversion, version, or postversion are in the scripts property of the package.json, they will be executed as part of running npm version.

Discussed offline, npm version will call preversion by default so this line will be executed when we run npm version